daemon/containerd/service.go

```diff
@@ -62,7 +64,10 @@ func (i *ImageService) GetLayerByID(cid string) (layer.RWLayer, error) {
 // LayerStoreStatus returns the status for each layer store
 // called from info.go
 func (i *ImageService) LayerStoreStatus() [][2]string {
-	return [][2]string{}
+	// TODO(thaJeztah) do we want to add more details about the driver here?
+	return [][2]string{
+		{"driver-type", "io.containerd.snapshotter.v1"},
```

Comment:
Or do we want to use the `plugins.SnapshotPlugin` const for this? https://github.com/containerd/containerd/blob/902212651b12e4d81b7bc779632b2e4da0a8e673/plugin/plugin.go#L65-L66

```go
	// SnapshotPlugin implements a snapshotter
	SnapshotPlugin Type = "io.containerd.snapshotter.v1"
```

Reply:
Ah, whoops, forgot to cast it to a string;

```
# github.com/docker/docker/daemon/containerd
..\..\daemon\containerd\service.go:70:19: cannot use plugin.SnapshotPlugin (constant "io.containerd.snapshotter.v1" of type plugin.Type) as type string in array or slice literal
```